src/commands/hardis/doc/project2markdown.ts

```diff
@@ -191,6 +193,7 @@ ${this.htmlInstructions}
       "- Code",
       "  - [Apex](apex/index.md)",
       "- [Lightning Pages](pages/index.md)",
+      "- [Approval Processes](approvalProcesses/index.md)",
```

Comment:
@MariiaPyvovarchuk please can you place Approval processes within Automations menu ? :)